zanata-maven-release

```diff
@@ -324,6 +324,6 @@ run_command sed -i -e 's|<tag>.*</tag>|<tag>HEAD</tag>|' $(find . -name pom.xml
 run_command ./mvnw "${MavenOptionArray[@]}" "${MavenTagProjects[@]}" versions:set -DnewVersion=$DevelopmentVersion
 run_command ./mvnw -N scm:checkin -Dmessage="chore(version): new snapshot version ${DevelopmentVersion}" -DpushChanges=true
 
-## Push the tags
-run_command git push --tags
+## Push the relevant tags
+run_command git push --follow-tags origin $WorkingBranch
```

Comment:
maven does not care the git tag. And we did remove and re-tag the 4.0.0, 4.1.0, 4.1.1

And this, does confuse some of our git clone.